settings.gradle

```diff
@@ -1,3 +1,4 @@
 // Copyright 2004-present Facebook. All Rights Reserved.
 
-include ':ReactAndroid', ':Examples:UIExplorer:android:app', ':Examples:Movies:android:app', ':Examples:TicTacToe:android:app'
+include ':ReactAndroid', ':Examples:UIExplorer:android:app', ':Examples:Movies:android:app', ':Examples:TicTacToe:android:app', ':processor'
+project(':processor').projectDir = new File('ReactAndroid/src/main/third-party/java/processor')
```

Comment:
I have thought where the processor module should be, two reasons not a top level module.
1. The processor module only work on android performance
2. The module should be a third-party library. The copy work with official source and another work alone version optimize custom modules and views created by app developers.